package.json

```diff
@@ -45,20 +45,22 @@
     "yargs": "^5.0.0"
   },
   "dependencies": {
-    "@dojo/core": "~0.3.0",
-    "@dojo/has": "~0.1.1",
+    "@dojo/core": "^0.3.1",
```

Comment:
We should be using `~` for internal dojo dependencies

Reply:
This doesn't matter for 0.x versions.

Reply:
I didn't want to create any friction. FWIW ~0.x.y = 0.x.y > 0.x+1.0 = ^0.x.y = 0.x.y > 0.x+1.0 just because zero is special in semver https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004